java/gtnh/egg-greg-tech--new-horizons.json

```
@@ -51,6 +51,26 @@
             "user_editable": true,
             "rules": "required|boolean",
             "field_type": "text"
+        },
+        {
+            "name": "Delete During Install \/ Reinstall",
+            "description": "These files will be deleted during install \/ reinstall",
+            "env_variable": "DELUPDATE",
```

I like speaking variables 
```suggestion
            "env_variable": "DELETE_ON_INSTALL",
```